pylint/checkers/classes/class_checker.py

```diff
@@ -192,7 +195,9 @@ def _is_trivial_super_delegation(function: nodes.FunctionDef) -> bool:
 # Deal with parameters overriding in two methods.
 
 
-def _positional_parameters(method):
+def _positional_parameters(
+    method: bases.BoundMethod | bases.UnboundMethod | nodes.FunctionDef,
```

Comment:
Why are these also methods? It is currently typed as `FunctionDef` in all its callers.

Reply:
You're right.